src/main/java/org/openelisglobal/common/rest/provider/bean/homedashboard/OrderDisplayBean.java

```diff
@@ -20,6 +20,8 @@ public class OrderDisplayBean {
 
     private String id ;
      
+    private String testSection;
```

Comment:
@rahul6603 can we use Integer testSecetionId instead of String testSection(name) ??